src/components/Modal/ModalManager.js

```diff
@@ -4,7 +4,7 @@ import Button from '../Button';
 
 import './Modal.scss';
 
-Modal.setAppElement('#root');
+if (process.env.NODE_ENV !== 'test') Modal.setAppElement('#root');
```

Comment:
Does CircleCI set `process.env.NODE_ENV` to "test"?

Reply:
When the command `yarn test` runs, it sets it to "test" apparently

Reply:
So the test will fail with `Modal.setAppElement('#root')`?

Reply:
Yes: https://github.com/reactjs/react-modal/issues/632